app/controllers/users_controller.rb

```diff
@@ -118,8 +118,13 @@ def edit
 
   # PUT /facilities/:facility_id/users/:id
   def update
+    previous = @user.price_groups.to_a
     @user_form = user_form_class.new(@user)
     if @user_form.update_attributes(edit_user_params) && @user.update_price_group(price_group_params)
+      if previous != @user.price_groups.reload.to_a
+        new_price_group = @user.price_groups.to_a - previous
```

Comment:
I'm thinking this variable should be plural since it's an array. `new_price_groups`.